lib/internal/bootstrap/node.js

```diff
@@ -426,5 +436,11 @@ function setupBuffer() {
       writable: true,
       configurable: true,
     },
+    'reportError': {
+      value: reportError,
+      enumerable: false,
+      writable: true,
+      configurable: true
+    },
```

Comment:
This should probably go in `lib/internal/bootstrap/browser.js` instead.

Reply:
Sure, no strong feelings about it but wdyt about the semantics/feature itself?

Reply:
It would make sense to also move atob/btoa (in a separate PR)?